[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = //div[@class="ivu-select-head-flex"]/input
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = //li[text()="金属（演示）"]
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = //button[contains(@class, "ivu-btn-primary")]
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = (//span[text()="系统管理"])[1]
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = (//span[text()="单元设置"])[1]
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = (//span[text()="环境设置"])[1]
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = //div[text()=" 周期 "]
[32mINFO    [0m root:base_page.py:26 查找元素：xpath = //div[label[text()="分派开始时间:"]]//input
[32mINFO    [0m root:base_page.py:26 查找元素：xpath = //div[label[text()="分派结束时间:"]]//input
[32mINFO    [0m root:base_page.py:26 查找元素：xpath = //div[label[text()="生产指令结束时间:"]]//input
[32mINFO    [0m root:base_page.py:26 查找元素：xpath = //div[label[text()="显示开始时间:"]]//input
[32mINFO    [0m root:base_page.py:26 查找元素：xpath = //div[label[text()="显示结束时间:"]]//input
[32mINFO    [0m root:base_page.py:26 查找元素：xpath = //div[label[text()="分派开始时间:"]]//input
[32mINFO    [0m root:base_page.py:26 查找元素：xpath = //div[label[text()="分派结束时间:"]]//input
[32mINFO    [0m root:base_page.py:26 查找元素：xpath = //div[label[text()="生产指令结束时间:"]]//input
[32mINFO    [0m root:base_page.py:26 查找元素：xpath = //div[label[text()="显示开始时间:"]]//input
[32mINFO    [0m root:base_page.py:26 查找元素：xpath = //div[label[text()="显示结束时间:"]]//input
[32mINFO    [0m root:base_page.py:43 点击元素：By = xpath, Value = //p[text()="保存"]
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000157026F7160>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/f77021515b839f3cf39107ccb0b1cf00
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000157026F7CD0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/f77021515b839f3cf39107ccb0b1cf00
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000157026F6BF0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/f77021515b839f3cf39107ccb0b1cf00
[32mINFO    [0m root:driver_manager.py:55 🛑 已从实例池移除并关闭 driver：1473214642256